django_project/user_profile/models/official.py

```diff
@@ -13,3 +15,16 @@ class Official(models.Model):
     phone = models.CharField(
         validators=[phone_regex], blank=True, max_length=15
     )
+    provider = models.OneToOneField(Provider, related_name='official_provider')
```

Comment:
@cchristelis 
as per discussion, we give just allowed form for provider
is my approach right? 

my idea is this user is representation of provider, for login and logout
and based on this user, we could give form for this provider